nixos/modules/services/misc/plex.nix

```
@@ -93,6 +93,15 @@ in
         '';
       };
 
+      accelerationDevices = mkOption {
+        type = types.listOf types.str;
+        default = [];
+        description = ''
+          A list of device paths to hardware acceleration devices that Plex should
+          have access to. This is useful for transcoding media files.
```

Escape hatches are important

```suggestion
          A list of device paths to hardware acceleration devices that Plex should
          have access to. This is useful when transcoding media files.
          The special value `"*"` will allow all devices.
```

This PR currently changes the default for existing setups, where no acceleration devices are used. At the very least it will require an entry into the release notes. Few read those however, and will then start complaining half year later about how plex "got worse over time".
I believe the default should be `[ "*" ]`, at the very least if the stateVersion is below 24.05.

---

Yeah, makes perfect sense. Maybe the default could be a subset of devices which are common, even going forward worth state version greater than 24.04

---

we just cross the "no more breaking changes" timelimit, so the cutouf would have to be 24.11.

---

Ok, made the default allow all, and added an example.